<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_Releve</class>
 <widget class="QDialog" name="D_Releve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Saisie des relevés de caisse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="3">
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OU</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Relevés précédents :</string>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remarques :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="3" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Saisir le contenu de la caisse :  </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="3">
       <widget class="QSpinBox" name="SBx_DixEuros"/>
      </item>
      <item row="5" column="3" rowspan="2">
       <widget class="QSpinBox" name="SBx_VingtEuros"/>
      </item>
      <item row="7" column="2" rowspan="2">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/50_euro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="SBx_CinquanteCents"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/5_euro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="SBx_DeuxEuros"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SBx_UnCent"/>
      </item>
      <item row="5" column="2" rowspan="2">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/20_euro.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QSpinBox" name="SBx_CinqEuros"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/20_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/2_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QSpinBox" name="SBx_DeuxCents"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="SBx_CinqCents"/>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="SBx_VingtCents"/>
      </item>
      <item row="2" column="2" rowspan="3">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/10_euro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/50_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/5_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/1_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3" rowspan="2">
       <widget class="QSpinBox" name="SBx_CinquanteEuros"/>
      </item>
      <item row="4" column="0" rowspan="2">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/10_cent.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QSpinBox" name="SBx_DixCents"/>
      </item>
      <item row="8" column="0" rowspan="2">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/1_euro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/2_euro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" rowspan="2">
       <widget class="QSpinBox" name="SBx_UnEuro"/>
      </item>
      <item row="9" column="2" rowspan="2">
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ImagesIcones.qrc">:/100_euro.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QSpinBox" name="SBx_CentEuros"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QLabel" name="label_21">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Différence par rapport au dernier relevé :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QDoubleSpinBox" name="SBx_Difference">
     <property name="minimum">
      <double>-9999999.990000000223517</double>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="2">
    <widget class="QTableView" name="TbV_Releves">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="8" column="9">
    <widget class="QLineEdit" name="Le_Remarques"/>
   </item>
   <item row="6" column="4" rowspan="2" colspan="3">
    <widget class="QLabel" name="label_20">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OU</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="8" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <widget class="QPushButton" name="Bt_Valider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Bt_PlusTard">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>0</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Plus tard dans 5 minutes</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QDoubleSpinBox" name="SBx_Total">
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Mois :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CBx_Mois"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Année :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="CBx_Annee"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
